tmt.spec

```
@@ -76,7 +76,9 @@ in a libvirt environment provisioned using testcloud.
 
 %package all
 Summary: Extra dependencies for the Test Management Tool
-Requires: tmt == %{version}-%{release}
+Requires: tmt >= %{version}
```

Well, actually, I'm not sure whether one wouldn't expect `version of tmt` = `version of tmt-all`.

IOW:
```suggestion
Requires: tmt = %{version}
```